### DISCUSSION TITLE
How to migrate to keepassxc from keepassx? With sharing.

### DISCUSSION THREAD

**Discussion by lonelyadmin** [keepassxreboot/keepassxc#9158] [Q&A] [answered]
Currently running keepass 2.53.1 on 2 computers. Database is password protected along with a key file. Database is stored on a network drive. 

I'd like to know the process for migrating to keepassxc from keepass without bringing any of the bad habits over, like sharing the master password, key file, and users having full access to everything. Having more than 2 users accessing the DB is desired, nothing large, maybe 4 or so. Ideally controlling access to certain content based on user is desired. Audit logs would be great per user. 

Currently we have things organized with many folders, which while not perfect, would match up with ACLs if that exists. I mention that because keeshare seems to work with directories (groups?) rather than individual entries or a whole database. My understanding may be completely wrong here. I tested a bit sharing the root folder and all directories were flattened/merged on the other machines. Is keeshare meant to be used per folder or group and not per database? I need to do more testing/learning/etc, but was wondering how others handle scenarios like this,

Thanks!

**Comment by droidmonkey** [comment, marked as answer]
If you want control like that you need a commercial solution. We implement thr keepass standard which is 100% meant to be a PERSONAL database. There is no such concept as ACL, audit logs, sharing, etc.

**Reply by lonelyadmin** [reply]
Thank you. Unfortunately those are features I have to have now.
